apm-protocol/apm-network/pom.xml

```diff
@@ -32,6 +32,11 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-codec-http2</artifactId>
+            <version>${netty.version}</version>
+        </dependency>
```

Comment:
You add a `netty dependency`, but you never exclude netty dependency from gRPC dependency. 
Also, grpc dependency is also existing in many other places.

<img width="990" alt="image" src="https://user-images.githubusercontent.com/5441976/182807529-e0e41d91-309f-4f7b-b97d-f215c2f0045e.png">

Especially agent-core pom.xml matters.

Reply:
This is still not resolved, there is another dependenc of gRPC in the pom.

Reply:
Is resolve, another dependency of gRPC , that not dependency netty-codec-http2 . but it dependency
netty-handler-proxy, that indirect dependency netty-codec-http2 for same version.